tests/unit/EmojiTest.js

```diff
@@ -25,40 +25,52 @@ describe('EmojiTest', () => {
         expect(emojiMatched).toBe(true);
     });
 
-    it('matches single emojis variants for size', () => {
-        // GIVEN an emoji that has the default Unicode representation WHEN we check if it's a single emoji THEN it should return true
+    it('matches only emojis variants for size', () => {
```

Comment:
```suggestion
    it('matches emojis for different variants', () => {
```

I don't think the size is relevant anymore now.

Reply:
added